awx/api/pagination.py

```diff
@@ -39,3 +56,19 @@ def get_html_context(self):
                                  for pl in context['page_links']]
 
         return context
+
+    def paginate_queryset(self, queryset, request, **kwargs):
+        from awx.api.views import UnifiedJobList
+        if isinstance(kwargs.get('view'), UnifiedJobList):
+            self.count_disabled = True
```

Comment:
This is maybe going to be controversial, but this `COUNT` is _very_ on the UJ endpoint is _very_ costly, and I don't see why we actually need it.

Reply:
![image](https://user-images.githubusercontent.com/214912/64652673-6ca7e680-d3f2-11e9-8eef-e1192dbe4d79.png)

Reply:
On second thought, @matburt, I don't think we can do this because the global jobs view seems to rely on it -_-

![image](https://user-images.githubusercontent.com/214912/64653657-f953a400-d3f4-11e9-843e-72a8e2bc3f37.png)